src/lib.rs

```diff
@@ -1245,9 +1247,14 @@ impl GooseAttack {
             );
         }
 
-        // Allocate a state for each of the users we are about to start.
         if !self.configuration.worker {
+            // Allocate a state for each of the users we are about to start.
             self.weighted_users = self.weight_task_set_users()?;
+
+            // Stand-alone and Manager processes can display metrics.
+            if !self.configuration.no_stats {
+                self.stats.display_metrics = true;
```

Comment:
Question: Do you also need to check for no_task_stats here?

Reply:
No, this flag only gets set to True on the Manager or a Stand-Alone Goose process. However, it made me realize a different recent commit was wrong, undoing it here:
https://github.com/tag1consulting/goose/pull/140/commits/19c88928a779db85c33cdbd893855823b305ffac#diff-6a9cab52f3c4564d1db3838cb39ed73fR395

(We can't ONLY check `display_metrics`, we also have to check if `tasks` is empty or we display empty task stats.)

While testing this I also realized we were allowing `--no-task-stats` to run on the Worker, which is incorrect -- it should only be allowed on the Manager, also fixed.